designer/client/src/Designer.tsx

```diff
@@ -31,7 +32,7 @@ function useData(props: Readonly<Props>): DataContextType {
    */
   const get = useCallback(async () => {
     const updated = await form.get(props.meta.id)
```

Comment:
Watch out here, the patched data goes to `setState()` but the unpatched data is returned

Can we make sure the cloned data from `updateConditions()` goes to both?

```suggestion
    const updated = updateConditions(await form.get(props.meta.id))
```

Reply:
Thanks, I missed that.  Resolved with ac0723e4e3f15eb77c049dff5c8274171b0ce4d0